Tutorials/G-CNN Honeycomb.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<p>can we have axes labels?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>of course</p>